app/components/mandatarissen/mandataris-summary.hbs

```diff
@@ -37,6 +37,16 @@
           <li class="au-c-list-horizontal__item">{{this.fractie}}</li>
         {{/if}}
       </ul>
+      <AuPill
+        @skin="link"
+        @draft={{false}}
+        @size="regular"
+        @icon="info-circle"
+        @iconAlignment="left"
+        @hideText={{false}}
```

Comment:
Not sure if I should rely more on default values here

Reply:
you probably could, but is okay.